.github/workflows/prebuild.yaml

```diff
@@ -0,0 +1,234 @@
+  CANVAS_PREBUILT_VERSION: "2.6.1"
+
+jobs:
+  Linux:
+    strategy:
+      matrix:
+        # Make sure to set this for the other builds also!
+        node: [8, 9, 10, 11, 12, 13, 14]
+    name: Node.js ${{ matrix.node }} on Linux
+    runs-on: ubuntu-latest
+    container:
+      image: chearon/canvas-prebuilt:7
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Build
+        env:
+          OS: ${{ runner.os }}
+        run: |
+          npm install -g node-gyp
+          npm install --ignore-scripts
+          . prebuild/$OS/preinstall.sh
+          cp prebuild/$OS/binding.gyp binding.gyp
+          node-gyp rebuild -j 2
+          . prebuild/$OS/bundle.sh
+
+      - name: Test binary
+        run: node -e "require('.')"
+
+      - name: Make bundle
+        id: make_bundle
+        run: . prebuild/tarball.sh
+
+      - name: Upload
+        uses: actions/github-script@0.9.0
+        with:
+          script: |
+            const fs = require("fs");
+            const assetName = "${{ steps.make_bundle.outputs.asset_name }}";
+            const tagName = `v${process.env.CANVAS_PREBUILT_VERSION}`;
+
+            const releases = await github.repos.listReleases({
+              owner: process.env.GITHUB_REPOSITORY.split("/")[0],
+              repo: process.env.GITHUB_REPOSITORY.split("/")[1]
+            });
+            const release = releases.data.find(r => r.tag_name === tagName);
+
+            for (const asset of release.assets) {
+              if (asset.name === assetName) {
+                await github.repos.deleteReleaseAsset({
+                  owner: process.env.GITHUB_REPOSITORY.split("/")[0],
+                  repo: process.env.GITHUB_REPOSITORY.split("/")[1],
+                  asset_id: asset.id
+                });
+                break;
+              }
+            }
+
+            // (This is equivalent to actions/upload-release-asset. We're
+            // already in a script, so might as well do it here.)
+            const r = await github.repos.uploadReleaseAsset({
+              url: release.upload_url,
+              headers: {
+                "content-type": "application/x-gzip",
+                "content-length": `${fs.statSync(assetName).size}`
+              },
+              name: assetName,
+              data: fs.readFileSync(assetName)
+            });
+
+  macOS:
+    strategy:
+      matrix:
+        node: [8, 9, 10, 11, 12, 13, 14]
+    name: Node.js ${{ matrix.node }} on macOS
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Build
+        env:
+          OS: ${{ runner.os }}
+        run: |
+          npm install -g node-gyp
+          npm install --ignore-scripts
+          . prebuild/$OS/preinstall.sh
```

Comment:
Scripts aren't using `$OS` so you could just change these to be `prebuild/${{runner.os}}` and delete the OS variable